keyboards/sol/readme.md

```diff
@@ -0,0 +1,10 @@
+Sol
```

Comment:
This file should be more aligned with the default keyboard readme template.

https://docs.qmk.fm/#/documentation_templates?id=keyboard-readmemd-template